### DISCUSSION TITLE
Can we use AHT10 High Precision I2C Digital Temperature And Humidity Measurement Module?

### DISCUSSION THREAD

**Discussion by Techposts** [pixelit-project/PixelIt#189] [Q&A]
I tried connecting the AHt10 but it seems it doesn't work. Is it possible to make it work with Pixelit to get the temp and Hum readinngs?

**Comment by o0shojo0o** [comment]
Technically, this is possible, but you need to find someone who owns the sensor and can integrate and test it.

**Reply by Techposts** [reply]
I tried the AHT10 and DHT20 (Digital I2C Sensors) They didn't work. I also used DHT11. Even that didn't got recognized / show any readings. DHT22 seems to work fine
